starters/blog/src/components/seo.js

```diff
@@ -18,7 +18,6 @@ const SEO = ({ description, lang, meta, title }) => {
           siteMetadata {
             title
             description
-            author
```

Comment:
Shouldn't be deleted. There is one usage here:

https://github.com/gatsbyjs/gatsby/blob/b25b52b51746831d051443fc7c66f682eacf7f6e/starters/blog/src/components/seo.js#L57-L60

Instead, query for `{ author { name } }` and replace the usage above with `site.siteMetadata.author.name`

Reply:
Well, I afraid that this is another bug. Should be `site.siteMetadata.social.twitter` as it used as attribution to the Twitter handle of the publisher. → #21773 

https://developer.twitter.com/en/docs/tweets/optimize-with-cards/guides/getting-started#card-and-content-attribution